tensorflow_addons/optimizers/weight_decay_optimizers_test.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for optimizers with weight decay."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+from tensorflow.python.framework import test_util
```

Comment:
let's use TF public API here.

Reply:
Thanks for the comments! I initially missed that the variable initializer is available via `compat.v1.global_variables_initializer()`. I'm now using that for the graph mode tests.  I've redirected `test_util` to the `addons.test_utils`, which still uses the private API.

Reply:
yes in `tfa.test_utils` there is a  `TODO: find public API alternative to these`